examples/wind_example.ipynb

```diff
@@ -0,0 +1,925 @@
+      "text/plain": [
+       "<Figure size 864x576 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "monthly_data = ndbc_data['2018']['2018-01']\n",
+    "daily_data = monthly_data['2018-01-10']\n",
```

Comment:
Because you need to repeat the month ('01') in the monthly data and daily data I would suggest making this yearly data then daily data or simply:
`daily_data = ndbc_data['2018']['2018-01-10']`

Reply:
Along with renaming these variables for clarity, I made Jan_data and Jan_11_data reference the yearly dataFrame directly